### PR TITLE
(es): Fix wrong translations in Spanish

### PR DESCRIPTION
**Pull request by jfernancordova**
Some words in spanish weren't translated correctly.

### CODE REVIEW

File: content/es/docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ weight: 30
...
 
-El diseño del Cloud Controller Manager está basado en un sistema de plugins, lo que permite a nuevos proveedores de servicios integrarse de forma fácil con Kubernetes. Se está trabajando en incorporar nuevos proveedores de servicios y para migrar los existentes del viejo modelo al nuevo CCM.
+El diseño del Cloud Controller Manager está basado en un sistema de plugins, lo que permite a nuevos proveedores de servicios integrarse de forma fácil con Kubernetes. Se está trabajando en implementar nuevos proveedores de servicios y para migrar los existentes del antiguo modelo al nuevo CCM.
 
-Este documento describe los conceptos tras el Cloud Controller Manager y da detalles sobre sus funciones asociadas.
+Este documento describe los conceptos tras el Cloud Controller Manager e informar detalles sobre sus funciones asociadas.
```
Comment (raelga) [inline content/es/docs/concepts/architecture/cloud-controller.md:13]:
```suggestion
Este documento describe los conceptos tras el Cloud Controller Manager y detalla sus funciones asociadas.
```

Comment (jfernancordova) [reply content/es/docs/concepts/architecture/cloud-controller.md:13]: Excelente @raelga!!!

**Comment by raelga** [comment]
Gracias por las correcciones!

/lgtm
/approve

**Pull request merged**